examples/transportation/segment/road/lanes/lanes-flow-reversible.yaml

```diff
@@ -25,8 +25,8 @@ properties:
               - access_type: allowed
                 when:
                   heading: forward
-                during: Mo-Su 00:00-12:00
+                  during: Mo-Su 00:00-12:00
```

Comment:
I don't love `during` here, it reads weird: "when heading is" vs "when during is". Would it make sense to rename this to "time"?